README.rst

```diff
@@ -21,89 +21,131 @@ PennyLane Cirq Plugin
     :alt: PyPI
     :target: https://pypi.org/project/pennylane-cirq
 
+.. image:: doc/_static/puzzle_cirq.png
```

Comment:
(Suggestion) 1. line renaming of `PennyLane Cirq` -> `PennyLane-Cirq`

Reply:
Sorry, where would that be? You mean line 22 (the ``:target:``)?

Reply:
Oh sorry about this! So on the first line of `README.rst`, I just couldn't comment there :sweat_smile:

Reply:
Perfect, done!